cmd/beaconchain/main.go

```diff
@@ -48,6 +48,9 @@ func main() {
 		beaconchain.SetSaveFile(*resetFlag)
 		bc = beaconchain.New(*numShards, *ip, *port)
 	}
+
+	fmt.Printf("Beacon Chain Started: /ip4/%s/tcp/%v/ipfs/%s\n", *ip, *port, bc.GetID().Pretty())
```

Comment:
Just some thinking. do you think we need to use log for all the printing and make it consistently over the whole code base?

Reply:
don't want to change this one now.  current beacon chain cmd will be obsolete soon by boot node.